src/Components/Facility/PatientNoteCard.tsx

```diff
@@ -173,7 +282,44 @@ const PatientNoteCard = ({
                 </div>
               </div>
             ) : (
-              <div className="text-sm text-gray-700">{noteField}</div>
+              <div
+                onClick={() => {
+                  if (allowThreadView) navigate(`${currentPath}/${note.id}`);
+                }}
+                className={`pl-11 text-sm text-gray-700 ${allowThreadView && "cursor-pointer"}`}
+              >
+                <MarkdownPreview markdown={noteField} />
+                <div className="flex gap-2">
+                  {note.files &&
+                    note.files.length > 0 &&
+                    note.files.map((file: any) => (
```

Comment:
1. A simpler check can be done by using `?` operator right?
2. Any type not allowed. Define proper types for the `files` property.

```suggestion
                  {note.files?.map((file) => (
```